application/lib/view/auth_page.dart

```diff
@@ -37,20 +61,24 @@ class AuthPage extends StatelessWidget {
     );
   }
 
-  Widget buildIdentifiersInput() {
-    return const Column(
+  Widget buildIdentifiersInput(BuildContext context) {
+    return Column(
       children: [
         AVAITextInput(
           label: 'E-mail institucional',
           placeholder: 'nome@ic.ufrj.br',
           allowNumbers: true,
           allowSpaces: false,
+          onChanged: (value) =>
+              Provider.of<AuthModel>(context, listen: false).setEmail(value),
```

Comment:
You're using `Provider.of<AuthModel>(context, listen: false)` two times, in this line and in line 81. You can easily transform `Provider.of<AuthModel>(context, listen: false)` into a variable and call the functions with it. For example:

```
const auth_provider = Provider.of<AuthModel>(context, listen: false);
auth_provider.setEmail(value);
```